HW1/task4.py

```diff
@@ -0,0 +1,15 @@
+# ребуется определить, можно ли от шоколадки размером n × m долек отломить k долек,
+#  если разрешается сделать один разлом по прямой между дольками 
+# (то есть разломить шоколадку на два прямоугольника).
+# *Пример:*
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input("введите чило n:"))
+m = int(input("введите чило m:"))
+k = int(input("введите чило k:"))
+
+if (k % n == 0) or (k % m == 0 ) and (1 <= k < n * m) :
```

Comment:
Во-первых, отдельные равенства иди неравенства не требует применения скобок. Однако скобки здесь все таки нужны.
```suggestion
if (k % n == 0 or k % m == 0 ) and 1 <= k < n * m:
```